src/app/features/public-form/public-form-config.ts

```diff
@@ -59,4 +59,10 @@ export class PublicFormConfig extends Entity {
 
   /** @deprecated use ColumnConfig directly in the columns array instead */
   @DatabaseField() prefilled: { [key in string]: any };
+
+  @DatabaseField({
+    label: $localize`:PublicFormConfig:Restricted Prefilled`,
+    isArray: true,
+  })
+  restrictedPrefilled: FieldGroup[];
```

Comment:
in what way are these "restricted"?

Reply:
my initial thought was that users should not be able to see this field if it's not available in columnConfig, so I considered the name "restrictedPrefilled." 
However, after thinking, "conditionalPrefilled" might be a better fit. This name indicates that the field is prefilled conditionally, based on specific configurations. 
whats your thought on this?

Reply:
maybe just "prefilledFields" in a very general way, then?